src/Dungeon.cpp

```diff
@@ -330,13 +330,13 @@ CMonsterDef *CDungeon::GetMonsterDef( int which_monster )
     return m_llMonsterDefs->GetLink( which_monster )->m_lpData;
 }
 
-bool CDungeon::SpawnMonster( int which_monster, JVector vSpawnPoint )
+bool CDungeon::SpawnMonster( int which_monster )
```

Comment:
so happy to not need this -- CMonster::CreateMonster() is (and has been) declared `static`, so no need to pass extra garbage through the call stack.